terraform/01-jenkins-server/main.tf

```diff
@@ -0,0 +1,80 @@
+provider "aws" {
+  profile = "default"
```

Comment:
For training purposes it's fine. But on real projects you will have multiple AWS profiles to work on different AWS accounts (NON PROD/PROD the simplest case), thus your will switch between accounts using ENV variable for instance.
For details https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication

Reply:
Profile moved to variables